Document how to update Agentless CloudFormation and Terraform versions (#26353)
DataDog/documentation: content/en/security/cloud_security_management/setup/agentless_scanning/cloudformation.md

{{% csm-agentless-exclude-resources %}}

## Update the CloudFormation stack

Datadog recommends updating the CloudFormation stack regularly, so you can get access to new features and bug fixes as they get released. To do so, follow these steps:

> 1. Log in to your AWS console and go to the CloudFormation Stacks page.
> 2. Select the `DatadogIntegration-DatadogAgentlessScanning-...` CloudFormation sub-stack, and click on `Update` then `Update nested stack`.
> 3. Click on `Replace existing template`.
> 4. In the following S3 URL: `https://datadog-cloudformation-template-quickstart.s3.amazonaws.com/aws/<VERSION>/datadog_agentless_scanning.yaml`, replace `<VERSION>` with the version found in [aws_quickstart/version.txt][4].
> 5. Paste the S3 URL and click on `Next` in the next pages, then submit the form.

Reviewer: ```suggestion
1. Log in to your AWS console and go to the CloudFormation Stacks page.
2. Select the **DatadogIntegration-DatadogAgentlessScanning-...** CloudFormation sub-stack, click **Update**, then click **Update nested stack**.
3. Click **Replace existing template**.
4. In the following S3 URL: `https://datadog-cloudformation-template-quickstart.s3.amazonaws.com/aws/<VERSION>/datadog_agentless_scanning.yaml`, replace `<VERSION>` with the version found in [aws_quickstart/version.txt][4]. Paste that URL.
5. Click **Next**, then submit the form.
```

Reviewer: Hey @kacembouhmadi , is there a name for the field users should paste the URL into in step 4? Also wondering if there are several pages users have to click "next" through at the end?

Author: The fiels is called "Amazon S3 URL" and yes, there are several pages that can be left unchanged

To uninstall Agentless Scanning, log in to your AWS console and delete the CloudFormation stack created for Agentless Scanning.

> ## Update the CloudFormation template version

Reviewer 2: Stack is probably more correct: 

```suggestion
## Update the CloudFormation stack
```

> To update the Agentless Scanning CloudFormation template:

Reviewer 2: ```suggestion
To update the Agentless Scanning CloudFormation stack:
```

> * Log in to your AWS console, select the `DatadogIntegration-DatadogAgentlessScanning...` sub-stack and click on `Update nested stack`

Reviewer 2: optional:
```suggestion
* Log in to your AWS console, select the `DatadogIntegration-DatadogAgentlessScanning...` CloudFormation sub-stack and click on `Update nested stack`
```

> * Log in to your AWS console, select the `DatadogIntegration-DatadogAgentlessScanning...` CloudFormation sub-stack and click on `Update nested stack`

Reviewer 3: I'd mention somewhere that this won't work if your version < 2.0.0. Updates on that earlier version are dangerous.

* Replace existing template using the following S3 URL:
  * `https://datadog-cloudformation-template-quickstart.s3.amazonaws.com/aws/<VERSION>/datadog_agentless_scanning.yaml`

>   * Replace `<VERSION>` in the URL above with the version found in https://github.com/DataDog/cloudformation-template/blob/master/aws_quickstart/version.txt

Author: Used your suggestion and added a bit more details.
I also moved the section higher in the page.
